ang/test/civicase/case/directives/case-overview.directive.spec.js

```diff
@@ -46,6 +48,35 @@
       });
     });
 
+    describe('Case Types', function () {
+      describe('when case type category filter is not present', function () {
+        it('filters the case types using case type category', function () {
```

Comment:
why is it that both scenarios end up with a `filters the case types using case type category` ? Shouldn't this one be something like `it displays all the case types` ?